guide/additional-info/changes-in-v14.md

```diff
@@ -531,6 +532,16 @@ A new `ignore` event has been added which is emitted whenever an element is not
 
 `CommandInteraction#commandGuildId` has been added which is the id of the guild the invoked application command is registered to.
 
+### ForumChannel
+
+Added class `ForumChannel` which represents GuildForum.
+
+# GuildForumThreadManager 
+Added `GuildForumThreadManager` as manager for threads in forum channels.
+
+# GuildTextThreadManager
+Added `GuildTextThreadManager` as manager for threads in text channels.
```

Comment:
These are not in alphabetical order.

Reply:
They should be in alphabetical order. It should be `GuildChannelManager` -> `GuildEmojiManager ` -> `GuildForumThreadManager` -> `GuildMemberManager` -> `GuildTextThreadManager`. You can correct the order too!

Reply:
K